lib/topological_inventory/ansible_tower/operations/source.rb

```diff
@@ -93,8 +109,10 @@ def update_endpoint(status, error_message)
 
           endpoint_update = SourcesApiClient::Endpoint.new
```

Comment:
Potentially move `endpoint_update` into its own method? `update_endpoint`?  Would allow us to test easier and be more consistent.

Reply:
Actually this is an `update_endpoint` method :)

Reply:
haha! It is.  I didn't see that the line numbers were skipped.